### PR TITLE
Don't comment when labeling PR size.

### PR DESCRIPTION
**Pull request by xiangpengzhao**
Fixes: #3933

/cc @fejta @krzyzacy 

and maybe 
/assign @foxish

### CODE REVIEW

File: mungegithub/mungers/size.go [hunk 1]
```diff
@@ -275,18 +266,7 @@ func calculateSize(adds, dels int) string {
...
-	}
-	stale := sizeRE.MatchString(*comment.Body)
-	if stale {
-		glog.V(6).Infof("Found stale SizeMunger comment")
-	}
-	return stale
-}
-
 // StaleIssueComments returns a slice of stale issue comments.
 func (s *SizeMunger) StaleIssueComments(obj *github.MungeObject, comments []*githubapi.IssueComment) []*githubapi.IssueComment {
```
Comment (cjwagner) [inline mungegithub/mungers/size.go]: This function can be removed completely and `RegisterStaleIssueComments(s)` can be removed from the `init` function.

**Comment by xiangpengzhao** [comment]
@cjwagner comments addressed. PTAL. Thanks!

**Comment by xiangpengzhao** [comment]
@cjwagner seems like this will break something as per https://github.com/kubernetes/test-infra/issues/3933#issuecomment-320843527. Any thoughts?

cc @cblecker

**Comment by cjwagner** [comment]
I guess we should avoid breaking the workflow if a lot of people use this? Though its gross that the munger creates and then quickly deletes comments just to trigger emails for labeling.
Something else to consider is that people who are not filtering out emails associated with these bot comments are getting spammed every time the bot labels a PR with a size label.

**Comment by fejta** [comment]
Personally I would strongly prefer to not have the comment spam

**Comment by xiangpengzhao** [comment]
>I guess we should avoid breaking the workflow if a lot of people use this? Though its gross that the munger creates and then quickly deletes comments just to trigger emails for labeling.

@thockin Tim, as you are using the workflow. WDYT?

>Something else to consider is that people who are not filtering out emails associated with these bot comments are getting spammed every time the bot labels a PR with a size label.

Yeah this is also my concern. Another concern is that some new contributors might be confused by bot's such behavior.

**Comment by cjwagner** [comment]
I think we should move forward with this if there are no objections since it will reduce email spam for many people without too much of an impact on those using the emails for labelling (the PR-size labels won't work any more, but that doesn't seem critical).
/lgtm

**Comment by kargakis** [comment]
SGTM

**Pull request merged**